integration/form-test.ts

```diff
@@ -783,6 +853,43 @@ test.describe("Forms", () => {
     });
   });
 
+  test.describe("with submitter button having `formMethod` attribute", () => {
+
+    test.describe("overrides the form `method` attribute with the button `formmethod` attribute", () => {
+      test("submits with GET instead of POST", async ({ page }) => {
+        let app = new PlaywrightFixture(appFixture, page);
+        await app.goto("/submitter-formmethod");
+        await app.clickElement("text=Submit with GET");
+        await page.waitForLoadState("load");
+        expect(await app.getHtml("pre")).toBe("<pre>GET</pre>");
+      });
+  
+      test("submits with POST instead of GET", async ({ page }) => {
+        let app = new PlaywrightFixture(appFixture, page);
+        await app.goto("/submitter-formmethod");
+        await app.clickElement("text=Submit with POST");
+        await page.waitForLoadState("load");
+        expect(await app.getHtml("pre")).toBe("<pre>POST</pre>");
+      });
+    })
+
+    test('uses the form "method" attribute', async ({ page }) => {
+      let app = new PlaywrightFixture(appFixture, page);
+      await app.goto("/form-method");
+      await app.clickElement("button");
+      await page.waitForLoadState("load");
+      expect(await app.getHtml("pre")).toMatch("POST");
+    });
+  
+    test('uses the button "formmethod" attribute', async ({ page }) => {
```

Comment:
```suggestion
    test('uses the button `formmethod` attribute', async ({ page }) => {
```

Reply:
fixed to also include add double, rather than single quotes around the full text